README.md

```diff
@@ -120,11 +124,13 @@ More options for `runner.py`
 
 4. If selections and output histogram/arrays need to be changed, modify the dedicated `workflows`
 
-5. Run the workflow with dedicated input and campaign name. Example commands for Run 3 can be found [here](#commands-for-different-phase-space). For first usage, the JERC file needs to be recompiled first, see [Create compiled JERC file](#create-compiled-jerc-filepklgz). You can also specify `--isArray` to store the skimmed root files.
+5. Run the workflow with dedicated input and campaign name. Example commands for Run 3 can be found [here](#commands-for-different-phase-space). For first usage, the JERC file needs to be recompiled first, see [Create compiled JERC file](#create-compiled-jerc-filepklgz). You can also specify `--isArray` to store the skimmed root files
 
-6. Once you obtain the `.coffea` file(s), you can make plots using the [plotting scripts](#plotting-code), if the xsection for your sample is missing, please add it to `src/BTVNanoCommissioning/helpers/xsection.py`
+6. Fetch the failed files to reprocess and processed events by with the `scripts/dump_processed.py`. Checked luminosity for procssed dataset used in the plot and run through failed jobs (details in [get procssed info](#get-processed-information))
```

Comment:
Fetch the failed files to obtain events that have been processed and events that have to be resubmitted using `scripts/dump_processed.py`. Check the luminosity of the processed dataset used for plotting and re-run failed jobs if needed (details in [get processed info](#get-processed-information)).